pkg/kubectl/cmd/annotate_test.go

```diff
@@ -118,8 +118,8 @@ func TestParseAnnotations(t *testing.T) {
 			expectErr:      false,
 		},
 		{
-			annotations:    []string{"url=" + testURL, api.CreatedByAnnotation + "=" + testJSON},
-			expected:       map[string]string{"url": testURL, api.CreatedByAnnotation: testJSON},
+			annotations:    []string{"url=" + testURL, api.TolerationsAnnotationKey + "=" + testJSON},
```

Comment:
This unit test verifies annotations with special characters work. As `CreatedByAnnotation` is removed, I substitute it with `TolerationsAnnotationKey`. I didn't change `testJSON` string variable as it will not affect this test case from passing. Should we change the `testJSON` variable accordingly?

Note that both `CreatedByAnnotation` and `TolerationsAnnotationKey` are in serialized JSON format.